coordinator/migrations/2023-09-04-090152_add_order_state_and_matches/up.sql

```diff
@@ -10,3 +10,22 @@ ALTER TABLE "orders" ADD COLUMN "order_state" "OrderState_Type" NOT NULL DEFAULT
 
 UPDATE orders SET order_state = 'Taken' WHERE taken = true;
 UPDATE orders SET order_state = 'Failed' WHERE taken = false;
+
+CREATE TYPE "MatchState_Type" AS ENUM (
+    'Pending',
+    'Filled',
+    'Failed'
+);
+
+CREATE TABLE "matches" (
+    id UUID PRIMARY KEY,
+    match_state "MatchState_Type" NOT NULL,
+    order_id UUID REFERENCES orders (trader_order_id) NOT NULL,
+    trader_id TEXT NOT NULL,
```

Comment:
Nit: 

could be a reference to user table

Reply:
I though about it, but it we unfortunately do not register the maker as user, so we can't reference here the user table as it would otherwise fail to create the matches for the maker.

Reply:
I created a ticket for it

https://github.com/get10101/10101/issues/1269